brevbaker/src/testFixtures/kotlin/no/nav/brev/brevbaker/IntegrationTestUtils.kt

```diff
@@ -117,6 +120,12 @@ fun <ParameterType : Any> Letter<ParameterType>.renderTestPDF(
     pdfFileName: String,
     path: Path = Path.of("build", "test_pdf"),
 ): Letter<ParameterType> {
+    if (!FeatureToggleSingleton.isInitialized) {
```

Comment:
Vet ikke helt om jeg er enig med meg selv med denne løsningen. Grunnen til det ble sånn er at brevbaker-modulen ikke har noe knytning til Unleash eller `FeatureToggleHandler`, men behovet er LetterExampleTest i pensjon-brevbaker-modulen. Motivasjonen for å ha det på denne måten er at man ikke trenger å tenke på feature toggles når det ikke har noe med det man skal teste.

Reply:
Synes dette er heilt kurant. Det er iallfall ganske i tråd med sånn eg tenkte da eg laga FeatureToggleSingleton.